include/ForceColl/Contact.h

```diff
@@ -150,6 +156,8 @@ class EmptyContact : public Contact
   {
     return "Empty";
   }
+
```

Comment:
Add a doxygen comment.

Reply:
Fixed in 5c962994a51049355a95d8ab509505676d082a25